### ISSUE TITLE
Lack of files

### ISSUE THREAD

**Issue by vdsukhov** [neurorestore/DE-analysis#1]
Hello!
I noticed that some scripts refer to files that are not included in the repository. Example:
file `DE-analysis/R/run_bulk_DE/outer-run-DE.R` has lines:
```R
source("R/functions/submit_job.R")
source("R/functions/detect_system.R")
```
Could you add this files to repository, please?

**Comment by skinnider** [comment]
Hi @vdsukhov, these are just functions used to run the jobs on the HPC clusters that we used in this work.
They are not necessary to reproduce the results and will almost certainly not work for you on your own HPC configuration.

**Comment by vdsukhov** [comment]
@skinnider thanks. So, basically I just need to call `run_DE` method from `Libra` package with data from `grid0` (variable from `DE-analysis/R/run_bulk_DE/outer-run-DE.R`)?

**Comment by skinnider** [comment]
Yes, that's what you would need to do if you were trying to reproduce our analysis. Are you trying to apply Libra to your own data? You will find a clearer description of how to do this at the Libra repository (https://github.com/neurorestore/Libra).

**Comment by vdsukhov** [comment]
@skinnider 
Yes, I'm trying to reproduce some of your analysis, especially repeat steps of differential expression analysis

**Comment by vdsukhov** [comment]
Can this function (`Libra::run_de`) be used for bulk datasets or it just works with single-cell data?

**Comment by skinnider** [comment]
For our paper, the code that was used for bulk datasets is in `R/analysis/run_bulk_DE`, and the relevant function is at https://github.com/neurorestore/DE-analysis/blob/master/R/functions/run_DE.R#L387.
